### ISSUE TITLE
Confusion about using commands

### ISSUE THREAD

**Issue by SyCode7** [anchore/anchore#53]
Hi, I recently installed anchore engine using docker-compose and the python command line client.
Reading through the documentation, i discovered another command pattern aside `anchore-cli ...` .  There are instructions using `anchore ....`Please are these different ? Also it seems the latter command is richer ??
Will be glad to get more clarity. Thanks.

**Comment by zhill** [comment]
@SyCode7 , what documentation are you referring to? Can you provide a link? We need to make sure this is clear for everyone. We know its a bit confusing.

There are two distinct CLIs:

_anchore_ - For executing local analysis of local docker images. This local tool has no API and stores results in local files only. It is used as a by the Anchore Engine code (which is why you find it inside the engine container), but generally shouldn't be used directly. These will be deprecated eventually as that functionality moves to a library-model instead of a cli tool model.

_anchore-cli_ - CLI for interacting with an anchore-engine installation via its HTTP API. Those commands are from a different package, [Anchore CLI](https://github.com/anchore/anchore-cli), and are what you see as 'anchore-cli'. If you are using Anchore Engine then that is the correct set of commands to use and the only commands that will interact with the service.

We will be deprecating things, renaming etc to make this more clear in the near future. I know its a source of confusion, particularly if you poke around inside the engine container.

Does that help clarify or still confused?

**Comment by nurmi** [comment]
Hi @SyCode7 

| Does the anchor-cli provide formatting options for scan results e.g json ?

It sure does!  you can use the '--json' option to anchore-cli for any command to view the raw API response in JSON, including policy evaluation:

```
# anchore-cli image add alpine
# anchore-cli image get alpine
<wait for the 'Analysis Status' to go to 'analyzed'>
# anchore-cli --json evaluate check alpine
```

There are lots of examples and walkthroughs on the [anchore-engine wiki]( https://github.com/anchore/anchore-engine/wiki) , plus the tool itself shows short help options:

```
# anchore-cli
Usage: anchore-cli [OPTIONS] COMMAND [ARGS]...

Options:
  --debug     Debug output to stderr
  --u TEXT    Username (or use environment variable ANCHORE_CLI_USER)
  --p TEXT    Password (or use environment variable ANCHORE_CLI_PASS)
  --url TEXT  Service URL (or use environment variable ANCHORE_CLI_URL)
  --insecure  Skip SSL cert checks (or use environment variable
              ANCHORE_CLI_SSL_VERIFY=<y/n>)
  --json      Output raw API JSON
  --version   Show the version and exit.
  --help      Show this message and exit.

Commands:
  evaluate      Policy evaluation operations
  image         Image operations
  policy        Policy operations
  registry      Registry operations
  repo          Repository operations
  subscription  Subscription operations
  system        System operations
```

**Comment by zhill** [comment]
@SyCode7 yep, @nurmi beat me to it. Happy to help and rest assured that we are working to make this less confusing.

**Comment by SyCode7** [comment]
@zhill  @nurmi thanks a lot guys. It works fine now. Exploring and enjoying this awesome tool, the 
`--details` and `--json`  switches provide very nice details. Will let you guys know if there are any issues or suggestions. 👍
